retrieval/target.go

```diff
@@ -88,11 +89,8 @@ func (t TargetHealth) value() model.SampleValue {
 }
 
 const (
-	// HealthUnknown is the state of a Target before it is first scraped.
```

Comment:
Isn't golint complaining if you remove all doc strings? I think the minimum required is a signle line above the `const` keyword.